src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationService registrationService;
+
+    @BeforeAll
+    public static void setUpAll() {
+        final StorageDao storage = new StorageDaoImpl();
+        User existingUser = new User();
+        existingUser.setAge(20);
+        existingUser.setLogin("existingUser");
+        existingUser.setPassword("password123");
+        storage.add(existingUser);
+    }
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_UserOk_notNull() throws ValidationException {
+        User user = new User();
+        user.setAge(20);
+        user.setLogin("username");
+        user.setPassword("password123");
+        User registeredUser = registrationService.register(user);
+        assertNotNull(registeredUser);
+        assertEquals(user, registeredUser);
```

Comment:
you can and should also check that that user has actually got to the `Storage.people`